cmake/windows-macros.cmake

```diff
@@ -232,19 +232,17 @@ if (WIN32 AND NOT BUILD_MSYS2_INSTALL)
         COMPONENT DTApplication)
   endif(LensFun_FOUND)
 
-  # Add iso-codes
-  if(IsoCodes_FOUND)
-    install(FILES
-        "${IsoCodes_LOCATION}/iso_639-2.json"
-        DESTINATION share/iso-codes/json/
-        COMPONENT DTApplication
-    )
-    file(GLOB IsoCodes_MO_FILES RELATIVE "${IsoCodes_LOCALEDIR}" "${IsoCodes_LOCALEDIR}/*/LC_MESSAGES/iso_639.mo")
-    foreach(MO ${IsoCodes_MO_FILES})
-      string(REPLACE "iso_639.mo" "" MO_TARGET_DIR "${MO}")
-      install(FILES "${IsoCodes_LOCALEDIR}/${MO}" DESTINATION "share/locale/${MO_TARGET_DIR}" COMPONENT DTApplication)
-    endforeach()
-  endif(IsoCodes_FOUND)
+  # Add .mo files
+  file(STRINGS ${CMAKE_SOURCE_DIR}/po/LINGUAS LINGUAS)
+  foreach(language in ${LINGUAS})
+    file(GLOB mofiles C:\\msys64\\mingw64\\share\\locale\\${language}\\LC_MESSAGES/*)
```

Comment:
Dunno about this one - are we guaranteed that files are there or can we dynamically check for them?

Reply:
`${CMAKE_SOURCE_DIR}/po/LINGUAS` is part of the source, so we can guarantee that.

`file(GLOB mofiles C:\\msys64\\mingw64\\share\\locale\\${language}\\LC_MESSAGES/*)` takes whatever mo files that msys2 has installed and uses them.  I guess if you could make another build system other than msys2 work, then the files wouldn't be there.  I could rework this and use the IsoCodes

Reply:
I just meant hardcoding `C:\\msys...` stuff. I don't remember if msys allows different location for their files during instal, but if it does we can't hardcode.

Reply:
Of course it does offer an arbitrary install location. On top of that, we don't only have the `mingw64` subsystem to think about , but also `ucrt64` and `clang64` (and `clangarm64`) in the not so distant future. I guess one could use the environment variables defined in /etc/msystem, like `MSYSTEM_PREFIX`. IMHO you probably don't need the full path since the installer is running from MSYS (i.e. `/`) anyway...

Reply:
It's changed, see latest commit

Reply:
Apologies, missed the "outdated" flag on the comments...